lib/flatpak-ref.h

```diff
@@ -71,4 +71,8 @@ FLATPAK_EXTERN char *        flatpak_ref_format_ref (FlatpakRef *self);
 FLATPAK_EXTERN FlatpakRef *   flatpak_ref_parse (const char *ref,
                                                  GError    **error);
 
+#ifdef FLATPAK_ENABLE_P2P
+FLATPAK_EXTERN const char *   flatpak_ref_get_collection_id (FlatpakRef *self);
+#endif  /* FLATPAK_ENABLE_P2P */
```

Comment:
Fixed and needing review in https://github.com/flatpak/flatpak/pull/884/commits/eddd2082919399204c0ab67308cc92bed8b15724, including making `FlatpakRef::collection-id` public.